_posts/2016-08-09-collections-app.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Collecting Connections through the Powerbox"
```

Comment:
The intended audience of the blog post seems to be fairly technical; given that, I'm +1 on this title.